src/GrassPokemon.java

```diff
@@ -0,0 +1,23 @@
+public class GrassPokemon extends Pokemon {
+    String food;
+    String quality;
+
+    public GrassPokemon(String name, int xp, String yell){
+        super(name, xp, yell);
+        super.setType("Grass");
+    }
+
+    @Override
+    public void chargingSound(){
+        System.out.println(super.getName() + " is rustling...");
+    }
+
+    public void leechSeed(){
```

Comment:
Je vertelt een leuk verhaal in deze methode, maar ook hier is de implementatie bijna hetzelfe in alle subklassen. Zo heb je alsnog veel copy/paste, wat je dus eigenlijk wilt voorkomen